src/App.tsx

```diff
@@ -1,18 +1,22 @@
-import { ThemeProvider, createTheme } from '@mui/material/styles';
-import { Route, Routes } from 'react-router-dom';
-import BottomNav from './components/BottomNav';
-import MainSection from './components/MainSection';
-import Navbar from './components/Navbar';
-import ActivitiesPage from './pages/ActivitiesPage';
-import ContactForm from './pages/ContactForm';
-import ContactsPage from './pages/ContactsPage';
-import SettingsPage from './pages/SettingsPage';
-import Wallet from './pages/Wallet';
+import { usePWA } from "@adorsys-gis/usepwa";
+import "@adorsys-gis/usepwa/dist/src/lib/components/main.css";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
```

Comment:
I'm not against importing the CSS file here, but this makes me think there is still a problem in the usePWA library. Can you update me on how you managed to fix CSS embedding from there?